ui/pages/settings/flask/view-snap/view-snap.js

```diff
@@ -109,138 +108,157 @@ function ViewSnap() {
+      paddingRight={3}
+    >
+      <Box
+        className="view-snap__header"
+        paddingTop={8}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <SnapAuthorship snapId={snap.id} />
+      </Box>
+      <Box
+        className="view-snap__description"
+        marginTop={4}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <Text variant={TextVariant.bodyMd} color={TextColor.textDefault}>
+          {snap.manifest.description}
+        </Text>
+      </Box>
+      <Box
+        className="view-snap__version_info"
+        marginTop={2}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <Text variant={TextVariant.bodyMd} color={TextColor.textDefault}>
+          {`${t('youInstalled')} `}
+          <span className="view-snap__version_info__version-number">
+            v{snap.version}
+          </span>
+          {` ${t('ofTextNofM')} `}
+          <ButtonLink
+            size={Size.auto}
+            href={url}
+            target="_blank"
+            className="view-snap__version_info__link"
           >
-            {snap.manifest.proposedName}
-          </Typography>
-          <Box className="view-snap__pill-toggle-container">
-            <Box className="view-snap__pill-container" paddingLeft={2}>
-              <SnapAuthorship snapId={snap.id} />
-            </Box>
-            <Box paddingLeft={4} className="view-snap__toggle-container">
-              <Tooltip interactive position="bottom" html={t('snapsToggle')}>
-                <ToggleButton
-                  value={snap.enabled}
-                  onToggle={onToggle}
-                  className="view-snap__toggle-button"
-                />
-              </Tooltip>
-            </Box>
-          </Box>
-        </div>
-        <Box
-          className="view-snap__install-details"
-          display="flex"
-          justifyContent="space-between"
-          alignItems="center"
-          padding={2}
-        >
-          {firstInstall && (
-            <Typography variant={TypographyVariant.H8}>
-              {t('snapAdded', [
-                formatDate(firstInstall.date, 'MMMM d, y'),
-                firstInstall.origin,
-              ])}
-            </Typography>
-          )}
-          <Typography
-            className="view-snap__version"
-            variant={TypographyVariant.H7}
+            {packageName}
+          </ButtonLink>
+          {` ${t('from').toLowerCase()} `}
+          <ButtonLink
+            size={Size.auto}
+            href={firstInstall.origin}
+            target="_blank"
+            className="view-snap__version_info__link"
           >
-            {t('shorthandVersion', [snap.version])}
-          </Typography>
-        </Box>
-        <Box
-          className="view-snap__content-container"
-          width={FRACTIONS.SEVEN_TWELFTHS}
+            {firstInstall.origin}
+          </ButtonLink>
+          {` ${t('on').toLowerCase()} ${formatDate(
+            firstInstall.date,
```

Comment:
Also if this string shows the latest version, we shouldn't populate the rest of the string with information from the first install

Reply:
Good catch. I updated the logic so it will take the last entry from an array that holds the version history and display these values from there. I also updated the terminology in the code, so it is now `installInfo`.